README.md

```diff
@@ -99,6 +99,123 @@ Run the following command:
 docker-compose up -d
 ```
 
+#### Docker Swarm w/ Docker Secrets:
+
+This section discusses running an instance of ibeam inside a locked docker swarm, and using the docker swarm facilities for managing secrets.
+Please refer to the following articles for in-depth details on docker swarm, locking the swarm, and using docker secrets.
+
+- [Swarm mode overview](https://docs.docker.com/engine/swarm/)
+- [Lock your swarm to protect its encryption key](https://docs.docker.com/engine/swarm/swarm_manager_locking/)
+- [Manage sensitive data with Docker secrets](https://docs.docker.com/engine/swarm/secrets/)
+
+Once you have a locked docker swarm instance initialized, you can create the secrets.
+On your host system create two secure (meaning not world-readable) files containing your Interactive Brokers account name and password:
+
+1. ib.account.txt
+2. ib.password.txt
+
+Next, inject these secrets into the docker swarm by using `docker secret create`:
+
+```posh
+docker secret create IBEAM_ACCOUNT_v1 ib.account.txt
+docker secret create IBEAM_PASSWORD_v1 ib.password.txt
+```
+
+Once you've initialized the secrets you should remove the original files from your host system.
+
+Create a `docker-compose.yml` file that references the new secrets, and that configures its environment to pull the secrets from the container's filesystem:
```

Comment:
Can this setup be done without docker-compose?

Reply:
Yes it can.  It's a bit involved (a long command line) but I've the added details on how to deploy IBeam as a service before the discussion of deploying it within a stack.

I wanted to note that my discussion of the firewall configuration was a result of trial and error as I am not having any luck finding documentation on its syntax.  As best as I can determine I've laid out how to set up a basic 'DENY FROM ALL / ALLOW FROM <specific_ips>' setup, but I'm not 100% confident due to the lack of documentation.  Do you have any pointers?

Reply:
Okay, cool, thanks for elaborating 👍

Reporting a discussion that brought you good results is perfectly fine! It's great you included these notes. If we figure out better ways to do it we'll be updating the docs accordingly. All good 😊